README.md

```diff
@@ -1,6 +1,80 @@
 # ink-wrapper
 
-## CI
+`ink-wrapper` is a tool that generates type-safe code for calling a substrate smart contract based on the metadata
+(`.json`) file for that contract.
+
+## Installation
+
+Install the tool from [crates.io](https://crates.io):
+
+```bash
+cargo install ink-wrapper
```

Comment:
Does it already work like that? If no, let's keep the `cargo install --git` and pass in the github link